playbooks/README.md

```diff
@@ -169,6 +169,52 @@ $ ansible-playbook -i inventory -e "selinux=permissive" playbooks/selinux.yml
 
 Be sure to update the [inventory file](../inventory) according to your OpenShift cluster configuration or use the file you used to deploy the cluster.
 
+### SR-IOV
```

Comment:
### Networking
Kubevirt-ansible enables sriov by default, using the kubernetes CNI plugin multus.
```yaml
# vars/all.yml
...
deploy_sriov_plugin: true
```

For sriov to work properly, configure your hosts to enable IOMMU, SR-IOV in the BIOS and kernel and vfio-pci in the kernel.   Also, enable the expected number of VFs before starting.

Reply:
@rthallisey please help me here, I am trying to understand if what you posted is a suggestion to replace some of the text below verbatim? Or is it something else?

Reply:
It was a suggestion to replace some text here in the in the README.